revscoring/languages/english.py

```diff
@@ -264,3 +266,23 @@ def safe_dictionary_check(word):
 :class:`~revscoring.languages.features.RegexMatches` features via a list of
 problematic words and phrases for use in reference text
 """
+
+
+def get_idioms():
+    session = mwapi.Session("https://en.wiktionary.org")
+
+    results = session.get(
+        action='query',
+        list='categorymembers',
+        cmtitle="Category:English idioms",
+        formatversion=2,
+        continuation=True)
+
+    idioms = []
+    for doc in results:
+        for page_doc in doc['query']['categorymembers']:
+            idiom = page_doc['title']
+            if 'Category:English' not in idiom:
```

Comment:
What's the meaning of this check?

Reply:
In the last five or so results.... There are values which are not idioms that have Category:English as suffixes. That's why I checked to ensure they are not added to the list.

Reply:
Aha!  That makes sense.  It might me work adding a comment explaining this.  Otherwise, it looks good.